ansible/vars.yml

```diff
@@ -8,8 +8,8 @@ postgres_major:
 
 # Full version strings for each major version
 postgres_release:
-  postgresorioledb-17: "17.0.1.053-orioledb"
-  postgres15: "15.8.1.060"
+  postgresorioledb-17: "17.0.1.054-orioledb"
```

Comment:
Ok, before we merge, let's add a suffix to these version numbers here like

```
postgresorioledb-17: "17.0.1.054-orioledb-etc-1"
```

Then sometime today we can test this in local infra, prior to merge.